code/modules/mob/living/carbon/carbon_life.dm

```diff
@@ -248,13 +248,9 @@
 /mob/living/carbon/handle_status_effects()
 	..()
 	if(stam_regen_start_time <= world.time)
-		if(stam_paralyzed)
-			update_stamina()
 		if(staminaloss)
-			setStaminaLoss(0, FALSE)
+			setStaminaLoss(0)
 			SEND_SIGNAL(src, COMSIG_CARBON_STAMINA_REGENERATED)
-			update_stamina_hud()
-			update_health_hud()
```

Comment:
Why remove these 2 update hud lines?

Reply:
they update in `setStaminaLoss(0)` without `FALSE` in second argument so we dont need to update it twice

Reply:
actually, health hud is not updated in update_stamina thing, but why would we update health bar since stamina has no affect on that anymore?